src/scripts/main.js

```diff
@@ -1 +1,21 @@
 'use strict';
+
+document.addEventListener('DOMContentLoaded', () => {
+  const gallery = document.getElementById('thumbs');
+  const largeImg = document.getElementById('largeImg');
+
+  // eslint-disable-next-line no-shadow
+  gallery.addEventListener('click', (event) => {
+    event.preventDefault();
+
+    const target = event.target.closest('a');
+
+    if (!target) {
+      return;
+    }
+
+    const newSrc = target.getAttribute('href');
```

Comment:
It would be beneficial to check if `newSrc` is not null or undefined before assigning it to `largeImg.src`. This would prevent potential errors if the `href` attribute is missing.